source/turtles.rst

```diff
@@ -157,7 +165,7 @@ Code in files
 
 Most code is written and executed from a file. 
 
-Using SublimeText create a file named `my_turtle_file.py` with this code:: 
+Using SublimeText to create a file named `my_turtle_file.py` with this code:: 
```

Comment:
I don't know what they have, I think we'll need to run through all of this on one of them in the edu centre